src/shared/components/LomFieldsInput/LomFieldsInput.tsx

```diff
@@ -56,6 +57,7 @@ const LomFieldsInput: FC<LomFieldsInputProps> = ({
 	subjectsPlaceholder,
 	themesPlaceholder,
 	limitDegreesByAlreadySelectedLevels = false,
+	filterSubjects = () => true,
```

Comment:
if the response is always true, then there is no point of having the filter function
```suggestion
	filterSubjects = () => boolean,
```

Reply:
This is the default value, typing is set to `filterSubjects?: (subject: Avo.Lom.LomField) => boolean;` above.